src/stratis_cli/_actions/_physical.py

```diff
@@ -18,16 +18,32 @@
 
 from __future__ import print_function
 
+from justbytes import Range
+
 from .._errors import StratisCliRuntimeError
 from .._errors import StratisCliUnimplementedError
 from .._stratisd_constants import StratisdErrors
 
 from ._connection import get_object
 from ._constants import TOP_OBJECT
+from ._constants import SECTOR_SIZE
 from ._data import ObjectManager
 from ._data import Pool
 from ._data import pools
+from ._data import MODev
+from ._data import devs
+from ._formatting import print_table
+
 
+def state_val_to_string(val):
+    """
+    Convert a blockdev state enumerated value to a string.
+    """
+    states = ["Missing", "Bad", "Spare", "Not-in-use", "In-use"]
```

Comment:
is it possible to, instead:

* Make ```states``` a module level constant.
* Omit this function entirely.
* Change table calculation as I suggest below.

?